plugins/cilium-cni/cni-install.sh

```diff
@@ -108,6 +111,30 @@ EOF
 EOF
 	;;
 
+"aws-cni")
+	cat > ${CNI_CONF_NAME} <<EOF
+{
+  "name": "aws-cni",
```

Comment:
I think we should start thinking on adding a `"version"` fields on these files.

Reply:
Can you be more concrete? Specify the CNI version? That creates its own problems as the plugins we are chaining with may support different versions. The CNI code translates between versions so we can currently chain old and new versions.

Reply:
Not the CNI version but a version of that configuration file in order for us to understand. If we start adding more functionalities how will we know if future cilium-cni still understand's today's configuration or vice-versa.

Reply:
We maintain 100% backward compatibility. What advantage does encoding a version have?